### DISCUSSION TITLE
node/node%E6%96%87%E6%A1%A3/Async%20hooks

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#790] [Announcements]
# node/node%E6%96%87%E6%A1%A3/Async%20hooks

Async hooks 好的，我们来聊一聊 Node.js 中的 Async Hooks。 Async Hooks 是 Node.js 的一个模块，用于跟踪异步资源的生命周期。在 Node.js 中，很多操作都是异步的，比如文件读写、网络请求等。这些异步操作在背后由事件循环(Event Loop)管理，而 Async Hooks 就提供了一种方法来观察...

https://doc.cherrychat.org/node/node%E6%96%87%E6%A1%A3/Async%20hooks.html

**Comment by syfssb** [comment]
hi
